### PR TITLE
Remove 9a

### PR DESCRIPTION
**Pull request by Lucyeoh**
**Remove**: 
```
9a) Privacy & Freedom of Expression:	
All projects must have strategies in place to anticipate, respond to and minimize adverse impacts on
privacy and freedom of expression where governments are believed to be using the project’s product
or services for illegitimate or political purposes.
```
**Reason**: This feels outside the scope of the OSS and more to do with the integrator, donor etc.

### CODE REVIEW

**Comment by lacabra** [comment]
👆 Cc: @eduffus, as the proposed changes are a result of a conversation with him, or rather an immediate reaction of trying to apply the current standard for vetting an active project.

**Comment by LawrenceHecht** [comment]
Privacy is relevant. Freedom of expression is not.  I read through the entire "The Age Digital Interdependence" report. It repeatedly mentions Human Rights and freedom of expression, but I don't believe it ever mentions those topics in conjunction with the digital goods topics. These are two separate topics that just happen to both be under the purview of the UN.

File: standard.md [hunk 1]
```diff
@@ -15,9 +15,8 @@ Indicator |	Requirement
 **7. Adherence to privacy and applicable laws** |	The project must state that it complies with relevant privacy laws, and all applicable international and domestic laws.
 **8. Adherence to standards & best practices** |	Projects must demonstrate some adherence to standards, best practices and/or principles. i.e. the principles for digital development
 **9. Do No Harm** | 	All projects must demonstrate that they have taken steps to ensure that the project anticipates, prevents and does no harm. 
-**9a) Privacy & Freedom of Expression** |	All projects must have strategies in place to anticipate, respond to and minimize adverse impacts on privacy and freedom of expression where governments are believed to be using the project’s product or services for illegitimate or political purposes.
```
Comment (sgoggins) [inline standard.md:18]: There are technical approaches to the development of digital public goods that advance this purpose more than others. Perhaps requiring a statement about how the project accomplishes this goal, and to what extent, would be useful for transparency. 

I agree that how digital public goods are used is not easily policed.

Comment (lacabra) [reply standard.md:18]: @sgoggins: I would like to ask you specifically what is your take on whether `strategies in place to anticipate, respond to and minimize adverse impacts on privacy and freedom of expression` is intrinsic to the development of digital public goods, or is an external factor tied to the implementation of that good out into the world. If it's the latter, it becomes a matter of policing which I would argue is out of the scope of this standard. On the other hand, if it is intrinsic, it should somehow be part of the standard.

What I am understanding from your `statement about how the project accomplishes this goal, and to what extent, would be useful for transparency` is something that seems to be under the purview of the the implementing party, and can be separated (completely?) from the actual digital public good.

**Comment by Lucyeoh** [comment]
In accordance with the new [governance.md](https://github.com/DPGAlliance/DPG-Standard/blob/master/governance.md) protocol below I am commenting on this issue to document and exploring the implications of accepting or rejecting the proposed change.  This should trigger a two week community input period that will end **November 5th, 2020**. 

**We invite you to leave your comments for or against the removal of 9a below.** 

```
Recommendation: Remove 9a. 
Reason: This feels outside the scope of the OSS and more to do with the integrator, donor etc. Designing for Privacy considerations inherent to the project, is already covered by 9b. 
```


_During this period we will inform the members of the Alliance's Internal Strategy Group (ISG) giving them an equal chance to comment. The final decision will require consensus from the [2 co-leads and 2 technical leads](https://github.com/DPGAlliance/DPG-Standard/blob/master/governance.md#current-roles)._ 

**Implications of accepting this recommendation and removing 9a:** 
* Narrows the scope of the standard to focus on the design of the project and not the implementation.
* Removes the implied assumption that project owners need to monitor and evaluate project implementations.
* Simplifies applications for projects without the capacity to control or even know about various implementations, for whom this question might have been a blocker.  
* Means we could include on our registry of digital public goods, a project that is being implemented in a way that adversely impacts freedom of expression and does harm. 

**Implications of rejecting this recommendation and keeping 9a:** 
*  Projects will be required to consider and potentially develop policies and practices for tracking implementations. 
* Some projects may be excluded that do not have the capacity or ability to understand their implementations. 
* May create tension and reduce the willingness of for digital public goods with large national implementations to participate in the registry. 

**We invite you to leave your comments for or against the removal of 9a below. This two week community input period will end **November 5th, 2020**. **

**Review by livmarte — APPROVED** [review]

**Review by christer-io — APPROVED** [review]

**Review by lacabra — APPROVED** [review]

**Pull request merged**